src/Hecke.jl

```diff
@@ -275,12 +265,16 @@ include("Deprecations.jl")
 #
 ################################################################################
 
+function ismaximal_order_known(K::AnticNumberField)
+  return has_attribute(K, :maximal_order)
+end
+
 function _get_maximal_order(K::AnticNumberField)
```

Comment:
I think nothing is left that is using `_get_maximal_order` (instead code just calls `maximal_order`), so perhaps it can be removed. Alternatively, one could change code to use `_get_maximal_order` again, to emphasize that precomputed data is accessed

Reply:
will think about it and adjust it, thanks.